sagas/mainSaga.js

```diff
@@ -1,17 +1,19 @@
 import { setTrendingMovies } from '../store/slices/moviesSlice';
 import { takeEvery } from 'redux-saga/effects'
+import { endpoints } from '../api/endpoints';
+import { initialize } from '../store/actions/initialize';
 
 function* fetchTrendingMovies() {
   console.log('ACTION CALLED')
-  // fetch('https://api.themoviedb.org/3/trending/all/day?api_key=d5826b4e12c757147537031e74238c63')
-  //   .then(response => response.json())
-  //   .then(data => {
-  //     dispatch(setTrendingMovies(data.results))
-  //   })
+  fetch(endpoints.trending)
+    .then(response => response.json())
+    .then(data => {
+      dispatch(setTrendingMovies(data.results))
+    })
```

Comment:
since you want to use sagas, is best if you use something like callApiSaga and do

```suggestion
  const response = yield call(FETCH_GET, endpoints.trending)
  yield put(setTrendingMovies(response.results)
```

or something like that, and make FETCH_GET a helper function similar to what we do for callApiSaga on the job.